src/EditorFeatures/Core.Cocoa/InlineRename/AbstractEditorInlineRenameService.cs

```diff
@@ -169,10 +169,11 @@ public async Task<IInlineRenameInfo> GetRenameInfoAsync(Document document, int p
                 {
                     var solution = document.Project.Solution;
                     var sourceDocument = solution.GetDocument(location.SourceTree);
-                    if (sourceDocument == null)
+                    Contract.ThrowIfNull(sourceDocument);
```

Comment:
💭 Any reason to not just use `GetRequiredDocument`?

Reply:
D'oh, fixed.